go/client/chat_svc_handler.go

```diff
@@ -727,15 +737,29 @@ func (c *chatServiceHandler) makePostHeader(ctx context.Context, arg sendArgV1,
 	return &header, nil
 }
 
-func (c *chatServiceHandler) getInboxLocalQuery(ctx context.Context, id chat1.ConversationID, channel ChatChannel) (chat1.GetInboxLocalQuery, error) {
+func (c *chatServiceHandler) getExistingConvs(ctx context.Context, id chat1.ConversationID, channel ChatChannel) ([]chat1.ConversationLocal, []chat1.RateLimit, error) {
+	client, err := GetChatLocalClient(c.G())
+	if err != nil {
+		return nil, nil, err
+	}
+
 	if !id.IsNil() {
-		return chat1.GetInboxLocalQuery{ConvID: &id}, nil
+		gilres, err := client.GetInboxAndUnboxLocal(ctx, chat1.GetInboxAndUnboxLocalArg{
+			Query: &chat1.GetInboxLocalQuery{
+				ConvID: &id,
+			},
+		})
+		if err != nil {
+			c.G().Log.Warning("GetInboxLocal error: %s", err)
+			return nil, nil, err
+		}
+		return gilres.Conversations, gilres.RateLimits, nil
 	}
 
 	// canonicalize the tlf name (no visibility necessary?)
 	tlfClient, err := GetTlfClient(c.G())
 	if err != nil {
-		return chat1.GetInboxLocalQuery{}, err
+		return nil, nil, err
```

Comment:
can you remove the (no visibility necessary) from 759?  i think i wrote that, but it's outdated.